Sources/core/objects/Projectiles.cpp~

```diff
@@ -0,0 +1,321 @@
+#include "Projectiles.h"
```

Comment:
This is a temporary file and it should be excluded from source tree. Run `git rm '*~'` in all directories with such files, then append line `*~` to .gitignore in root of repository so git will not add them again.